pycbc/detector.py

```diff
@@ -122,8 +122,9 @@ def light_travel_time_to_detector(self, det):
         d = self.location - det.location
         return float(d.dot(d)**0.5 / constants.c.value)
 
-    def antenna_pattern(self, right_ascension, declination, polarization, t_gps):
+    def antenna_pattern(self, right_ascension, declination, polarization, t_gps, pol_flag= 'Tensor'):
```

Comment:
@srashtig I think this mostly looks great, but can you change the 'pol_flag' option to something like 'polarization_type'? And presumably one should be able to select between the vector, tensor, etc independently. Otherwise, I think this is largely good to go, please have a look at the code climate warnings though.